app/assets/javascripts/custom.js

```diff
@@ -126,3 +79,65 @@ function openNav() {
 	}
 
   
+function showListWiseGraph(id, index){
+    var isExpanded = $("#collection" + index).attr("aria-expanded");
+    if(isExpanded === 'true'){
+        return;
+    }
+    
+    $.getJSON('/static_pages/show', { collection_id: id}, function(data) {
+        render_graph(data);
+    })    
+}
```

Comment:
We can pass functions by name if they have the same arity, right? Can't we write this like this:

```javascript
$.getJSON('/static_pages/show', { collection_id: id}, render_graph)
```

I might be wrong about that, through - my knowledge of JS is limited.